x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/trusted_apps_page.tsx

```diff
@@ -14,30 +14,29 @@ import { TrustedAppsNotifications } from './trusted_apps_notifications';
 import { CreateTrustedAppFlyout } from './components/create_trusted_app_flyout';
 import { getTrustedAppsListPath } from '../../../common/routing';
 import { useTrustedAppsSelector } from './hooks';
-import { getListCurrentShowValue, getListUrlSearchParams } from '../store/selectors';
+import { getCurrentLocation } from '../store/selectors';
 
 export const TrustedAppsPage = memo(() => {
   const history = useHistory();
-  const urlParams = useTrustedAppsSelector(getListUrlSearchParams);
-  const showAddFlout = useTrustedAppsSelector(getListCurrentShowValue) === 'create';
+  const location = useTrustedAppsSelector(getCurrentLocation);
```

Comment:
var name `location` could get confusing since react-router has a `useLocation()` hook. consider another name maybe?

Reply:
Will fix it in my next set of commits.